doc/source/examples/analysis/alignment_and_rms/pairwise_rmsd.ipynb

```diff
@@ -63,21 +64,37 @@
    "metadata": {},
    "source": [
-    "## Pairwise RMSD"
+    "## Background\n",
+    "\n",
+    "While [1-dimensional RMSD](rmsd.ipynb) is a quick way to estimate how much a structure changes over time, it can be a misleading measure. It is easy to think that two structures with the same RMSD from a reference frame are also similar; but in fact, they can be very different. Instead, calculating the RMSD of each frame in the trajectory to all other frames in the other trajectory can contain much more information. This measure is often called the pairwise, all-to-all, or 2D RMSD.\n",
+    "\n",
+    "The other, or reference, trajectory in pairwise RMSD can either be the first trajectory, or another one. If the pairwise RMSD of a trajectory is calculated to itself, it can be used to gain insight into the conformational convergence of the simulation. The diagonal of the plot will be zero in this case (as this represents the RMSD of a structure to itself). Blocks of low RMSD values *along* the diagonal indicate similar structures, suggesting the occupation of a given state. Blocks of low RMSD values *off* the diagonal indicate that the trajectory is revisiting an earlier state. Please see the living guide [Best Practices for Quantification of Uncertainty and Sampling Quality in Molecular Simulations by Grossfield et *al.*](https://github.com/dmzuckerman/Sampling-Uncertainty) for more on using 2D RMSD as a measure of convergence. MDAnalysis provides a [DistanceMatrix](#Pairwise-RMSD-of-a-trajectory-to-itself) class for easy calculation of the pairwise RMSD of a trajectory to itself.\n",
+    "\n",
+    "When the other trajectory in pairwise RMSD is a different trajectory, the pairwise RMSD can be used to compare the similarity of the two conformational ensembles. There is no requirement that the two trajectories be the same length. In this case, the diagonal is no longer necessarily zero. Blocks of low RMSD values anywhere indicate that the two trajectories are sampling similar states. [Pairwise RMSDs with different trajectories must be manually calculated in MDAnalysis](#Pairwise-RMSD-between-two-trajectories)."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Pairwise RMSDs are an effective way to quickly view similarities and differences in conformations (as measured by RMSD) across a trajectory.\n",
+    "## Pairwise RMSD of a trajectory to itself"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "While 1-dimensional RMSD is a quick way to estimate how much a structure changes over time, it can be a misleading measure. It is easy to think that two structures with the same RMSD from the reference frame are also similar; but in fact, they can be very different. Instead, calculating the RMSD of each frame in the trajectory to each other frame in the reference can contain much more information. This measure is often called the pairwise, all-to-all, or 2D RMSD.\n",
```

Comment:
COnsidering the really awesome background bit you wrote above, I don't know if this section is necessary anymore, thoughts?

Reply:
Oops, I forgot to delete that!